package.json

```diff
@@ -5,8 +5,10 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc -w",
+    "build:grammar": "./antlr/gradlew -p ./antlr generateGrammarSource",
```

Comment:
This line doesn't work for me on windows, but if we change it to `bash ./antlr/gradlew -p ./antlr generateGrammarSource` it does work. I had @kjmahalingam test as well and it still worked for him with this new command, so we may want to change this.

Reply:
OK.  I can make the change.  I wondered about that -- but assumed it was probably ok because I thought I had done something similar in the past.  Oh well.